app/main/views/sign_in.py

```diff
@@ -97,11 +97,10 @@ def _get_access_token(code, state):  # pragma: no cover
     try:
         access_token = response_json["access_token"]
     except KeyError as e:
-        # Capture the response json here so it hopefully shows up in error reports
         current_app.logger.exception(
-            f"Error when getting access token {response.json()} #notify-admin-1505"
+            f"Error when getting access token {response_json()} #notify-admin-1505"
```

Comment:
It looks like these are still being changed, but I think it was intentional to refer to `response.json()` in this context because there was a key being looked for that didn't exist.  The error in this case is showing the original response content so that someone could verify this in the logs and check to see what's happening or missing.

```suggestion
            f"Error when getting access token {response.json()} #notify-admin-1505"
```